cipher_suite_test.go

```diff
@@ -21,3 +21,28 @@ func TestDecodeCipherSuites(t *testing.T) {
 		// todo: compare result
 	}
 }
+
+func TestCipherSuiteName(t *testing.T) {
+	testCases := []struct {
+		suite    uint16
+		expected string
+	}{
+		{0xc0ac, "TLS_ECDHE_ECDSA_WITH_AES_128_CCM"},
+		{0x0000, "0x0000"},
+	}
+
+	for _, testCase := range testCases {
+		res := CipherSuiteName(testCase.suite)
+		if res != testCase.expected {
+			t.Fatalf("Expected: %s, got %s", testCase.expected, res)
+		}
+	}
+}
+
+func TestAllCipherSuites(t *testing.T) {
```

Comment:
This is an extremely silly test. Doesn't seem like it would ever catch anything, but it keeps the coverage up :man_shrugging:

Reply:
`if actual == 0 { t.Fatalf() }` may be enough for this purpose?

Reply:
Yap, seems fair. Changed it to that now.